neuronav/agents/README.md

```diff
@@ -6,25 +6,13 @@ These consist of implementations of the canonical Q-Learning, Actor-Critic, Valu
 
 All algorithms are "tabular" and work with observations that consist of integer representations of the state of the agent. This corresponds to the `index` observation type.
```

Comment:
I would rewrite this as:
The algorithms included the beta release* are "tabular".  Tabular algorithms work with observations that are integer representations of the state of the agent (e.g., which grid the agent is in a grid world). This corresponds to the `index` observation type.
* Future releases will include algorithms and environments that support rich observation.